classes/01class/exercises/c01-aws01/mrcsmonteiro/README.md

```diff
@@ -0,0 +1,294 @@
+# C01-AWS01
+
+## Commands Execution Output
+
+- Commands to create the first EC2 instance and any additional resource required:
+```
+$ aws ec2 run-instances \
```

Comment:
Although it will work, are you able to create this EC2 instance with a more secure security group?

Reply:
Added a SG that allows only my IP address to connect via SSH.